biom/table.py

```diff
@@ -2935,6 +2921,161 @@ def _intersect_id_order(self, a, b):
                 idx += 1
         return new_order
 
+    def concat(self, others, axis='sample'):
```

Comment:
Not blocking, but just making an observation, this method is a little bit long, is there any way to break it down a bit?